cmake/Scripts/configure

```diff
@@ -48,6 +48,7 @@ Optional Packages:
   --with-superlu=PATH     user defined path to SuperLU library
   --with-umfpack=PATH     use UMFPACK/SuiteSparse from a specified location
   --with-ert=PATH         Use ERT libraries
+  --with-zoltan=PATH      Use ZOLTAN libraries
```

Comment:
> It is perhaps a little unfair to require this now

Sure.  However, I never looked at the script until now so I didn't realise the problem existed before.

> Which path is required, is it the "prefix" path (e.g. `/usr/local`) or the full path (e.g. `/usr/local/lib`)?

"It depends".

The `configure` script massages the data and sets the CMake cache variable `${pkgname}_ROOT` (in this case, `ZOLTAN_ROOT`) to the canonical version of whatever is supplied as `PATH`.  The `FindZOLTAN.cmake` module then uses that variable in whichever way it deems appropriate.

I haven't checked to see if passing just `--with-zoltan` (i.e., no `=PATH`) does the right thing, but from a cursory reading of the `configure` script I don't think it will.

> Is this behaviour uniform across the `PATH` arguments

Sort of.  Like I said, the `configure` script is basically a front-end to `cmake` and just sets cache variables (like `${package}_ROOT`) on the `cmake` command line.